packages/web-react/src/hooks/useValidationText.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode, useMemo } from 'react';
+import { ValidationState, ValidationTextType } from '../types';
+
+export interface UseValidationTextProps {
+  messageClassName?: string;
+  validationState?: ValidationState;
+  validationText?: ValidationTextType;
+}
+
+export const useValidationText = (props: UseValidationTextProps): ReactNode => {
+  const { messageClassName, validationState, validationText } = props;
+
+  return useMemo(() => {
+    if (!(validationState && validationText)) {
+      return;
+    }
+
+    const getMessage = (message: string, index?: number) => (
```

Comment:
What about having a `ValidationText` component?

Reply:
I created `useValidationText`

Reply:
I mean instead of `getMessage`:

```
validationText.map((text, index) => <ValidationText key={...}>{text}</ValidationText>)
```

🤷 But nevermind.